src/content/reference/manifest.mdx

```diff
@@ -60,6 +63,7 @@ Each image supports the following fields:
 - `args`: add build arguments, which are environment variables accessible only during the build process. Build arguments with a value containing a `$` sign are resolved to the environment variable value on the machine okteto is running on.
 - `export_cache`: image tag for exported cache when build.
 - `cache_from`: list of images to import cache from.
+- `depends_on`: list of images that need to be built first.
```

Comment:
What happens if there's a dependency cycle? Do we detect it? (Image A depends on B, and image B depends on A)

Reply:
Sorry for the late response. Yes, we detect it and throw a proper error